### ISSUE TITLE
Possible Xbox 360 console support?

### ISSUE THREAD

**Issue by IcicaSora** [passinglink/passinglink#27]
Since there really isn't any cheap(er) X360 fightboard alternatives and there exist some information on the Internet about the auth process between the controller and the console, wonder if adding xbox 360 support is doable (or viable on something like the bluepill).

**Comment by dogtopus** [comment]
No, because the pre-shared (?) secret for xbox360 controller auth is still not public so passthrough seems to be the way to go. However Zephyr still doesn't have USB host support and third party, portable, open source USB host stack is quite an unobtainium in practice.

**Comment by aenertia** [comment]
TinyUSB is quite usable at this point and used in several similar projects https://github.com/hathach/tinyusb

Check out i.e : https://github.com/sanjay900/Ardwiino

**Comment by JStyle21** [comment]
It's true TinyUSB now supports Zephyr as an RTOS
https://github.com/zephyrproject-rtos/zephyr/issues/42066#issuecomment-2786993999
